### DISCUSSION TITLE
Bat as Lister replacement

### DISCUSSION THREAD

**Discussion by bkaradzic** [sharkdp/bat#2958] [Ideas]
I was trying to use Bat as replacement for Lister (Orthodox File Manager concept) in Total Commander / Krusader, but I ran into a few issues.

1. Bat can't be closed with ESC key.

2. Binary files are not displayed as hex, as-in how it would be displayed by typical hex viewer.

3. Configuring pager is kind of unintuitive. I managed to make it work for code with `--paging always --pager "less -RF"` but it wasn't really consistent for all types of files.

Any plans to add these features?

**Comment by keith-hall** [comment]
> 1. Bat can't be closed with ESC key

You can use a pager which does allow it, or configure `less`:
https://unix.stackexchange.com/q/183457/152147

> 2\. Binary files are not displayed as hex, as-in how it would be displayed by typical hex viewer.

A wrapper script or perhaps [something like `rsop` could help here](https://github.com/sharkdp/bat/issues/800#issuecomment-987855856)

> 3\. Configuring pager is kind of unintuitive. I managed to make it work for code with `--paging always --pager "less -RF"` but it wasn't really consistent for all types of files.

I guess we'd need more information to really answer this one, but there were some ideas about how to integrate a custom pager into `bat`: https://github.com/sharkdp/bat/issues/1053
